src/04-modality-specific-files/29-motion.md

```diff
@@ -0,0 +1,224 @@
+| RotationOrder              | RECOMMENDED           | string        | Specify the sequence in which the elemental rotations are applied around the extrinsic axes. A 3D rotation is represented about 3 different axes (Tait-Bryan convention). Must be one of: “XYZ”, ”XZY”, ”YXZ”, “YZX”, “ZXY”, “ZYX”.                                                                                                                                                                                                                                                                                              |
+| SamplingFrequencyEffective | REQUIRED              | number        | Effective sampling rate of the tracking system in Hz. If duration of the corresponding recording is available, effective sampling rate is computed by dividing the `RecordingDuration` in seconds by the number of samples included in the time series. If not available, the field takes the same value as field `SamplingFrequency`.                                                                                                                                                                                           |
+| SoftwareVersions           | RECOMMENDED           | string        | Manufacturer’s designation of the acquisition software.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| SpatialAxes                | RECOMMENDED           | string        | Refers to the coordinate system in which the motion data are to be interpreted, if the recorded data can be mapped to a fixed reference frame. A sequence of characters F/B (forward-backward), L/R (left-right), and U/D (up-down). The position of a character in the sequence determines which of the X,Y,Z axes it maps to. For example, “FRD” for  X-forward,  Y-right, Z-down. For 1D or 2D cases, only specify the used axes and use the character “_” for unused axes (“F_R” when the Y axis is not used, for instance). |
+| SubjectArtefactDescription | RECOMMENDED           | string        | Freeform description of the observed subject artefact and its possible cause (e.g., “stopped at 20 min”, ”fall at 10 min”). If this field is left empty, it will be interpreted as absence of artifacts.                                                                                                                                                                                                                                                                                                                         |
```

Comment:
would this be related to the derivatives annotations?

Reply:
Ah! I see, the EEG specs also have this field, then it makes sense to have it here too if it's useful. Once annotations specs are available, we can see how they relate.